alf/networks/encoding_networks.py

```diff
@@ -298,17 +302,17 @@ def __init__(self,
             input_size = input_tensor_spec.shape[0]
 
         self._fc_layers = nn.ModuleList()
-        if fc_layer_params is None:
-            fc_layer_params = []
+        if _fc_layer_params is None:
+            _fc_layer_params = []
         else:
-            assert isinstance(fc_layer_params, (tuple, list))
```

Comment:
I mean just change this to `isinstance(fc_layer_params, tuple)` and problem solved.

Reply:
Now enforcing all ```xxx_params``` to be tuples.